### PR TITLE
AP_Periph: populate the param meta data for the wiki

### PR DESCRIPTION
**Pull request by magicrub**
Follow-up for https://github.com/ArduPilot/ardupilot_wiki/pull/3448

### CODE REVIEW

File: Tools/AP_Periph/Parameters.cpp [hunk 1]
```diff
@@ -43,26 +43,57 @@ extern const AP_HAL::HAL &hal;
 const AP_Param::Info AP_Periph_FW::var_info[] = {
     // @Param: FORMAT_VERSION
     // @DisplayName: Eeprom format version number
-    // @Description: This value is incremented when changes are made to the eeprom format
+    // @Description: This value represents the current version number of the eeprom struture format. Set to zero to erase eeprom and perform a factory reset.
```
Comment (Hwurzburg) [inline Tools/AP_Periph/Parameters.cpp:46]: spelling structure

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: Tools/AP_Periph/Parameters.cpp (hunk 1 above)
Comment (magicrub) [reply Tools/AP_Periph/Parameters.cpp]: fixed. I changed the text back to match the other vehicles

**Comment by magicrub** [comment]
@Hwurzburg everything else ok? Ready to merge?

**Review by khancyr — COMMENTED** [review]
Could we add this to CI testing like other?

**Comment by magicrub** [comment]
@Hwurzburg I merged this, hope you don't mind. I wanted to make sure it's not holding up the wiki PR. We can adjust this in another PR if need-be.

**Comment by Hwurzburg** [comment]
thanks....did not realize there were so many additional params to the libraries, but guess it makes sense since you have to setup the serial ports,etc.I will use this to extract them for special attention in the base doc...

**Comment by magicrub** [comment]
@Hwurzburg yeah, it would be nice to have library-only params but there needs to be a few one-offs. At some point it would be nice to bring in the serial manager library to handle the baud and port stuff.

File: Tools/AP_Periph/Parameters.cpp [hunk 2]
```diff
@@ -47,22 +47,53 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
...
 
+    // @Param: BRD_SERIAL_NUM
+    // @DisplayName: Serial number of device
+    // @Description: Non-zero positive values will be shown on the CAN App Name string
+    // @Range: 0 2147483648
+    // @User: Advanced
     GSCALAR(serial_number, "BRD_SERIAL_NUM", 0),
 
 #ifdef HAL_PERIPH_ENABLE_BUZZER_WITHOUT_NOTIFY
+    // @Param: BUZZ_VOLUME
```
Comment (Hwurzburg) [inline Tools/AP_Periph/Parameters.cpp:90]: @magicrub  Tom, need to change this to BUZZER_VOLUME not emitting properly

**Pull request merged**